internal/cli/atlas/livemigrations/options/live_migrations_opts.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 type LiveMigrationsOpts struct {
+	cli.OutputOpts
```

Comment:
@colm-quinn If you are adding it here it is better to remove it from the create commands as well

Reply:
Nice catch! Will do!